core/src/main/java/io/grpc/NameResolverProvider.java

```diff
@@ -108,6 +108,16 @@ private static ClassLoader getCorrectClassLoader() {
     return Thread.currentThread().getContextClassLoader();
   }
 
+  protected static boolean isAndroid() {
```

Comment:
Because it was protected in the other class for OkHttp to decide if it was more important than Netty. Swapped to private.